v5/server/rox_options.go

```diff
@@ -1,32 +1,35 @@
 package server
 
 import (
+	"time"
+
 	"github.com/rollout/rox-go/v5/core/logging"
 	"github.com/rollout/rox-go/v5/core/model"
-	"time"
 )
 
 type RoxOptionsBuilder struct {
-	Version                     string
-	DevModeKey                  string
-	FetchInterval               time.Duration
-	Logger                      logging.Logger
-	ImpressionHandler           model.ImpressionHandler
-	ConfigurationFetchedHandler model.ConfigurationFetchedHandler
-	RoxyURL                     string
-	SelfManagedOptions          model.SelfManagedOptions
-	DynamicPropertyRuleHandler  model.DynamicPropertyRuleHandler
+	Version                     	string
+	DevModeKey                  	string
+	FetchInterval               	time.Duration
+	Logger                      	logging.Logger
+	ImpressionHandler           	model.ImpressionHandler
+	ConfigurationFetchedHandler 	model.ConfigurationFetchedHandler
+	RoxyURL                     	string
+	SelfManagedOptions          	model.SelfManagedOptions
+	DynamicPropertyRuleHandler  	model.DynamicPropertyRuleHandler
+	NetworkConfigurationsOptions	model.NetworkConfigurationsOptions
 }
 
 type roxOptions struct {
-	version                     string
-	devModeKey                  string
-	fetchInterval               time.Duration
-	impressionHandler           model.ImpressionHandler
-	configurationFetchedHandler model.ConfigurationFetchedHandler
-	roxyURL                     string
-	selfManagedOptions          model.SelfManagedOptions
-	dynamicPropertyRuleHandler  model.DynamicPropertyRuleHandler
+	version							string
+	devModeKey						string
+	fetchInterval					time.Duration
+	impressionHandler				model.ImpressionHandler
+	configurationFetchedHandler		model.ConfigurationFetchedHandler
+	roxyURL							string
+	selfManagedOptions				model.SelfManagedOptions
+	dynamicPropertyRuleHandler		model.DynamicPropertyRuleHandler
+	networkConfigurationsOptions	model.NetworkConfigurationsOptions
```

Comment:
I wonder whether this will create a compile error. It could do with go fmt running over it anyawy

Reply:
i swear it's a straight line in my IDE, tried to fix it twice
i'll try edit it here over github
🤷

Reply:
i edited it over github, and then my VSCode showed it in offside
seems github and VSCode doesn't agree on how `tab` should space things
i right clicked on VSCode and `format` which changed it to spaces, now it looks the same in both